tools/perf/scripts/bench_run_log.py

```diff
@@ -237,7 +243,16 @@ def single_run(self):
         # Log the payload $step_count times
         with Timer() as timer:
             for _ in range(self.num_steps):
-                run.log(payload)
+                if not self.is_unique_payload:
+                    run.log(payload)
+                else:
+                    run.log(PayloadGenerator(
+                        self.data_type, self.num_metrics, self.metric_key_size
+                    ).generate())
+                
+                if self.time_delay_second > 0:
```

Comment:
leave a comment that this is a hackaround the issue with the 500s that you were seeing

Reply:
Will do.

Just to be clear, I was thinking to keep it even if we don't have the 500s and 502s errors for high # of metrics.

There are times when I have to reproduce real-life workloads that usually have some time in between their logging steps before the next metric is computed and ready to log.  Just want to be prepared with this flag.